app/services/change_schedule.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+class ChangeSchedule
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Validations::Callbacks
+
+  attribute :cpd_lead_provider
+  attribute :participant_id
+  attribute :reason
+  attribute :course_identifier
+  attribute :schedule_identifier
+  attribute :cohort
+
+  alias_attribute :user_profile, :participant_profile
```

Comment:
I was wondering why we needed to alias participant profile 🤔, could we always instead use `participant_profile` to avoid confusion ?